articles/azure-developer-cli/reference.md

```diff
@@ -915,24 +1001,23 @@ Manage templates.
 
 List templates.
 
-```azdeveloper
+```
 azd template list [flags]
 ```
 
 ### Options
 
-```azdeveloper
--h, --help            Gets help for list.
--o, --output string   The output format (the supported formats are JSON, table; the default is table).
+```
+  -h, --help            Gets help for list.
+  -o, --output string   The output format (the supported formats are json, table). (default "table")
```

Comment:
Keep previous version.

Reply:
The behavior of generating this text is defined by a Go library and can't be easily overridden.